### ISSUE TITLE
import custom queries from DB

### ISSUE THREAD

**Issue by arxdsilva** [prest/prest#739]
## Problem
When having multiple instances of `prestd`, it's now only possible to have custom queries from a folder, this way you'd have to share them and instantiate on every new container.

## Desired behavior
Be able to create, load and manage scripts directly from the installed DB.

## Possible solution
New flag with default to folder loading, if not creates a table named `prestd_custom_queries` that allows creating custom queries and loading them on app start directly from the DB.

**Comment by vmesel** [comment]
This meets something I was talking with @avelino. We need a separate application database to handle auth and this custom queries.

I'm not a fan of having pREST and my own data on the same database.

**Comment by arxdsilva** [comment]
> This meets something I was talking with @avelino. We need a separate application database to handle auth and this custom queries.
> 
> I'm not a fan of having pREST and my own data on the same database.

Im certainly a +1 on this

**Comment by vmesel** [comment]
@arxdsilva I agree with that! I was even thinking today of data lineage and data dictionary functionalities on the codebase, so we could let people source through the data as best as they can with all of the open info.

One thing that is a pain for my team is knowing where we can find certain data on a database. Yesterday I was trying to make a simple join with one ID and another, but I wasn't able to find which table had both so I could connect them.

Getting back to DB management, is there an issue open about that?

**Comment by arxdsilva** [comment]
> Getting back to DB management, is there an issue open about that?
@vmesel I think this should fit: https://github.com/prest/prest/issues/327
